content/chapters/io/lab/solution/client-server/client.c

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: BSD-3-Clause
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
+#include "utils/utils.h"
+
+#define PORT	5000
+#define BUFSIZE	1024
+
+const char *localhost = "127.0.0.1";
+
+int readInputWithPrompt(char *buf)
+{
+	char *ret;
+
+	printf("Message for receiver (type 'exit' to quit): ");
+
+	memset(buf, 0, BUFSIZE);
+	ret = fgets(buf, BUFSIZE, stdin);
+	if (!ret)
+		return -1;
+	buf[strlen(buf)-1] = '\0';
+
+	return 0;
+}
+
+void populateSockAddr(struct sockaddr_in *sockaddr, const char *addr, int port)
+{
+	int ret;
+
+	sockaddr->sin_family = AF_INET;
+	sockaddr->sin_port = htons(port);
+	ret = inet_aton(addr, (struct in_addr *) &sockaddr->sin_addr.s_addr);
+	DIE(ret == 0, "inet_aton");
+}
+
+int main(void)
+{
+	char buf[BUFSIZE];
+	int ret, sockfd;
+	struct sockaddr_in server_addr;
+
+	while (1) {
+		// Get user input as a string.
```

Comment:
```suggestion
		/* Get user input as a string. */
```
Use C-style comments.